contributors.yml

```diff
@@ -1 +1,2 @@
 - mcansh
+- SCasarotto
```

Comment:
Should be lowercase for some reason 🤷‍♂️

```suggestion
- scasarotto

```

Reply:
ah shoot, i fixed this in the inverse 😂